components/dashboard/tailwind.config.js

```diff
@@ -21,23 +41,42 @@ module.exports = {
                 teal: colors.teal,
                 sky: colors.sky,
                 rose: colors.rose,
-                "gitpod-black": "#161616",
+                "gitpod-black": podkitColors.black,
                 "gitpod-red": "#CE4A3E",
                 "kumquat-dark": "#FF8A00",
                 "kumquat-base": "#FFAE33",
                 "kumquat-ripe": "#FFB45B",
                 "kumquat-light": "#FFE4BC",
                 "kumquat-gradient": "linear-gradient(137.41deg, #FFAD33 14.37%, #FF8A00 91.32%)",
-                "gray-900": "#12100C",
-                "gray-800": "#23211E",
-                "gray-700": "#514F4D",
-                "gray-600": "#565451",
-                "gray-500": "#666564",
-                "gray-400": "#999795",
-                "gray-300": "#DADADA",
-                "gray-200": "#ECE7E5",
-                "gray-100": "#F5F4F4",
-                "gray-50": "#F9F9F9",
+                gray: podkitColors.gray,
+                // Podkit colors - eventually we'll only use these colors
+                // Once migrated, we can remove the colors above and shift this up under theme directly instead of extend
+                "pk-content": {
```

Comment:
First step in setting up the new colors for the design system. This will let us reference contextual class names in our podkit components.